test-e2e/build/has-build-time-deps.test.js

```diff
@@ -3,13 +3,15 @@
 const childProcess = require('child_process');
 const path = require('path');
 
-const {initFixture} = require('../test/helpers');
+const {initFixture, skipSuiteOnWindows} = require('../test/helpers');
+
+skipSuiteOnWindows("#272");
 
 describe('Build - has build time deps', () => {
 
   let p;
 
-  beforeAll(async () => {
```

Comment:
I needed to change some of these `beforeAll` to `beforeEach`. The problem is that the `beforeAll` was being run, even though the suite was 'skipped' - and in some cases, this step would fail. Using `beforeEach` defers execution until the tests are actually run.

It may have a minimal performance impact, but for commands like `esy build`, they tend to be much faster the second time around since most of the work is done on the initial execution.